src/setup.sh

```diff
@@ -273,7 +288,34 @@ function get_env_file {
 }
 
 #
-# Checks that this script can be run on the current machine and exits with an 
+# Checks if HomeBrew is installed.
+#
+function is_brew_installed {
+  type brew > /dev/null 2>&1
+}
+
+#
+# Checks if MacPorts is installed.
+#
+function is_port_installed {
+  type port > /dev/null 2>&1
+}
+
+#
+# Checks if all MacPorts requirements are met.
+#
+function check_port_prerequisites {
+  if ! type boot2docker > /dev/null 2>&1 ; then
+    log_error "MacPorts-based setup expects Docker Compose to be preinstalled."
```

Comment:
Looks like a copy/paste error. This error message should be about Boot2Docker and the one below should be about Docker Compose.